[DOCS-12567] Updating inconsistent policy names (#428)
mulesoft/docs-api-manager: modules/ROOT/pages/rate-limiting-sla-policy.adoc

=== Choosing Between a Cluster or a Standalone Configuration

> If you use only one Client application as an authorization mechanism in your server, the same recommendations as in the Rate Limiting policy applies to your use case. You might have several Mule instances and you might map different client applications to different servers in the load balancer.

Reviewer: ```suggestion
If you use only one client application as an authorization mechanism in your server, the same recommendations as in the Rate Limiting policy apply to your use case. You might have several Mule instances and you might map different client applications to different servers in the load balancer.
```

Author: will fix

In a worst case scenario, the number of penalized requests with latency due to cluster consistency is constant and independent from the actual size of the configured quota. Consequently, the smaller the window, the greater the percentage of potentially delayed requests. 

Therefore, you must set up window sizes greater than one minute in Rate Limiting and Rate Limiting - SLA-based policy configurations only for a cluster scenario.

=== Choosing Persistence for Your Rate Limiting - SLA-based Policy

> You can configure your Rate Limiting - SLA-based policy to use windows that persist as long as days, months, and years. For example, suppose you want to allow your user to consume 1 million requests per year, but you cannot ensure that the node will be up the entire period or will need maintenance, which may result in restarting the Mule runtime engine.

Reviewer: ```suggestion
You can configure your Rate Limiting - SLA-based policy to use windows that persist as long as days, months, and years. For example, suppose you want to enable your user to consume 1 million requests per year, but you cannot ensure that the node will be up the entire period or will not need maintenance, which might result in restarting Mule.
```

Author: will update

=== Configuring Policy Parameters for Anypoint Service Mesh (Non-Mule Applications)

> The Rate Limiting - SLA-based policy works in the same way for Anypoint Service Mesh (non-Mule applications), excluding the following differences:

Reviewer: ```suggestion
The Rate Limiting - SLA-based policy works in the same way for Anypoint Service Mesh (non-Mule applications), but with the following differences:
```

Author: will fix
